server/controllers/entities/lib/properties/properties_values_constraints.coffee

```diff
@@ -31,14 +31,20 @@ module.exports =
   'wdt:P31': _.extend {}, bases.uniqueEntity, { adminUpdateOnly: true }
   # author
   'wdt:P50': bases.humanEntity
+  # founded by
+  'wdt:P112': bases.humanEntity
   # publisher
   'wdt:P123': bases.uniqueEntity
+  # owned by
+  'wdt:P127': bases.entity
   # movement
   'wdt:P135': bases.entity
   # genre
   'wdt:P136': bases.entity
   # based on
   'wdt:P144': bases.workEntity
+  # headquarters location
+  'wdt:P159': bases.entity
```

Comment:
this should probably also be let aside (as for `wdt:P17`) as we can't properly validate a location entity at the moment